dev-docs/bidders/aniview.md

```diff
@@ -5,13 +5,17 @@ description: Prebid Aniview Bidder Adapter
 pbjs: true
 biddercode: aniview
 media_types: banner, video
+gpp_supported: true
+ortb_blocking_supported: true
+multiformat_supported: true
 tcfeu_supported: true
 floors_supported: true
 usp_supported: true
 schain_supported: true
 safeframes_ok: true
 gvl_id: 780
 sidebarType: 1
+userIds: true
```

Comment:
This is not a boolean. `userId: (list of supported vendors)`

If you support reading every possible EID, you may say 'all' here.